### PR TITLE
SmartThings Traces Test Data

### PR DESCRIPTION
**Pull request by beckje01**
Added another special case trace.
Added description file for explinations of the traces.

### CODE REVIEW

File: zipkin-ui/testdata/testdata-explination.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+smartthings-oauth-authoization.json
```
Comment (codefromthecrypt) [inline zipkin-ui/testdata/testdata-explination.md]: nit typo in the file enclosing this explination/explanation

**Review by codefromthecrypt — COMMENTED** [review]
thanks and good idea. It might be fine to just append to the README instead though? could be easier for discovery..wdyt?

**Comment by beckje01** [comment]
I'll redo this into the readme I was thinking it should go there but wasn't sure.

**Comment by codefromthecrypt** [comment]
thanks much!

**Comment by codefromthecrypt** [comment]
Thank you SOOO much it is great to have a big trace like this. We've been waiting a while to have data like this!

once "lens" is out, we can try to add screenshots of trace and/or dependency graph to the README cc @openzipkin/ui

**Pull request merged**